pkg/kubelet/config/file_linux.go

```diff
@@ -151,3 +154,27 @@ func (s *sourceFile) consumeWatchEvent(e *watchEvent) error {
 	}
 	return nil
 }
+
+func modifyStaticPodForPinnedManagement(pod *v1.Pod) {
```

Comment:
We can filter the pods based on the label `io.openshift.management: true`

Reply:
Thanks. Updated.